packages/iov-ethereum/src/serialization.ts

```diff
@@ -124,4 +201,20 @@ export class Serialization {
       return fromHex(normalizeHex(encodeQuantity(checkedNonce.toNumber())));
     }
   }
+
+  /**
+   * Value 0 must be represented as 0x instead of 0x0 for some strange reason
```

Comment:
Source?

Reply:
Unfortunatly I have no source for this. Just rial and error. Otherwise the calculated public key is wrong. Value 0 was never tested before. But this is at least consistent with the nonce case above.